EIPS/eip-2628.md

```diff
@@ -0,0 +1,41 @@
+---
+eip: eip-2628
+title: Header in StatusMessage
+author: Tomasz K. Stanczak (@tkstanczak)
+discussions-to: TBD
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-04-29
+---
+
+## Simple Summary
+
+This document proposes to include the whole BlockHeader in the status message and change the way the header hash is calculated to decrease the required size of the StatusMessage in such scenario.
+
+## Abstract
+
+Remove number, bestHash, and genesisHash and include the full block header in the Eth62 StatusMessage. When calculating a header hash use Keccak(Bloom) instead of Bloom when serializing header.
+
+## Motivation
+
+Currently after a handshake with a new node, if the node has a better total difficulty block, then we need to ask for the block header to be able to verify the total difficulty claim.
+With this change this will be no longer required which will make the network slightly more resilient to sybil attacks.
```

Comment:
I don't really see why sending a header is more secure then sending number/hash. You can still create a fake header for free, mine it with insignificant PoW and say that it's header 20 million with TD infinite. You don't gain any security by sending more data, or at least I don't see how.